go/vt/vttablet/onlineddl/executor.go

```diff
@@ -765,8 +766,93 @@ func (e *Executor) terminateVReplMigration(ctx context.Context, uuid string) err
 	return nil
 }
 
+func (e *Executor) killQueriesOnTable(ctx context.Context, tableName string) error {
```

Comment:
This isn't really covered by any tests, is it? This feels like the most critical aspect as we're killing things off in the production system.

Things like this are actually easier in unit tests as you can mock the mysql query responses.

Reply:
This is covered in https://github.com/vitessio/vitess/pull/14546/files#diff-e1236744a601a269891381b0ed09b7151d86f5a9b001d8c4e954211c2ae04a8d ; there's a test that holds an open transaction, we attempt a completion, the migration does not complete; we then force cut-over, and we see that the migration completes.